README.md

```diff
@@ -1,42 +1,31 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+Mobile App VERSION
+Project Name
+MOBILE-VERSION
 
-# Project Name
+ Project Description.
 
->my-portfolio  
+Built With
+Major languages HTML and CSS
+Frameworks
+Stylelint e.t.c
 
+👤 Author1
+ADEBOWALE ADEGBOYE
+GitHub: @ademibowale
+Twitter: @ademibowale1
+LinkedIn: https://www.linkedin.com/in/adebowale-adegboye-143568221/
 
-## Built With
-
-- Major languages HTML and CSS
-- Frameworks
-- Stylelint e.t.c
-
-## Authors
-
-👤 *Author1*
-
-- GitHub: [@adebowale](https://github.com/adebowale)
-- Twitter: [@adebowale](https://twitter.com/adebowale)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/adebowale)
-
-
-
-## 🤝 Contributing
-
+🤝 Contributing
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
+Feel free to check the issues page.
 
+Show your support
 Give a ⭐ if you like this project!
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] I see that you are using the MIT file so it would be better if you keep the `License` section

Reply:
Implemented 👍